fiesta/apps/buddy_system/templatetags/buddy_system.py

```diff
@@ -79,3 +81,21 @@ def get_matched_buddy_requests(context):
         responsible_section=request.membership.section,
         state=BuddyRequest.State.MATCHED,
     ).order_by("-matched_at")
+
+
+@register.filter
+def get_color_by_text(name: str) -> str:
+    hash_object = hashlib.md5(name.encode())
```

Comment:
pybandit security scanner dislikes this, the `usedforsecurity=False` should be fine for us


```
>> Issue: [B324:hashlib] Use of weak MD5 hash for security. Consider usedforsecurity=False
   Severity: High   Confidence: High
   CWE: CWE-327 (https://cwe.mitre.org/data/definitions/327.html)
   More Info: https://bandit.readthedocs.io/en/0.0.0/plugins/b324_hashlib.html
   Location: fiesta/apps/buddy_system/templatetags/buddy_system.py:88:18
   ```
   
```suggestion
    hash_object = hashlib.md5(name.encode(), usedforsecurity=False)
```